packages/Takeover/src/Takeover.js

```diff
@@ -1,35 +1,44 @@
 import React from "react";
 import PropTypes from "prop-types";
+import { zValue } from "@paprika/stylers/lib/helpers";
 import Overlay from "@paprika/overlay";
 import extractChildren from "@paprika/helpers/lib/extractChildren";
 import FocusLock from "./components/FocusLock";
-import * as styled from "./Takeover.styles";
+import * as sc from "./Takeover.styles";
 
 const propTypes = {
-  /** The content for the Takeover. */
+  /* Description of the Takeover dialog for assistive technology */
+  a11yText: PropTypes.string,
+
+  /** The content for the Takeover */
   children: PropTypes.node.isRequired,
 
-  /** Control the visibility of the takeover */
+  /** Control the visibility of the Takeover */
   isOpen: PropTypes.bool.isRequired,
 
-  /** Callback triggered when the takeover needs to be close */
-  onClose: PropTypes.func,
+  /** Callback once the Takeover has been closed event */
+  onAfterClose: PropTypes.func,
 
-  /** Callback once the takeover has been opened event */
+  /** Callback once the Takeover has been opened event */
   onAfterOpen: PropTypes.func,
 
-  /** Callback once the takeover has been closed event */
-  onAfterClose: PropTypes.func,
+  /** Callback triggered when the takeover needs to be close */
+  onClose: PropTypes.func,
+
+  /** Z-index of the Takeover content */
+  zIndex: PropTypes.number,
 };
 
 const defaultProps = {
-  onAfterClose: () => {},
+  a11yText: null,
   onClose: () => {},
+  onAfterClose: () => {},
   onAfterOpen: () => {},
+  zIndex: zValue(5),
```

Comment:
Why 5? 🤷‍♂️

Reply:
Also, I don't consider this a breaking change, but it's possible that it _could_ affect how an app renders a view with the Takeover.